packages/vscode-extension/README.md

```diff
@@ -13,7 +13,7 @@ The following software is required to work with SuiteCloud Extension for Visual
 
 Read the detailed list of prerequisites in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installation Requirements for SuiteCloud Extension for Visual Studio Code</a>.
 
-## <font size="6"> <span style="color:gray"> Installing SuiteCloud Extension for Visual Studio Code </font> </span>
+## <font size="6"> <span style="color:gray"> Installing SuiteCloud Extension for Visual <br/> Studio Code </font> </span>
```

Comment:
This one looks wrong or at least different from the other ones.
That said, I don't know if adding `</br>` break line is a "smart" fix, and how it will result in different screen sizes/resolutions.

Reply:
I agree on the different resolutions... but this is all that came to my mind so far. Fixing the break, although it also works.